src/bin/wallet.rs

```diff
@@ -3,6 +3,7 @@ extern crate bincode;
 extern crate bs58;
```

Comment:
Adding this to lib makes any of the ones in `bin/` redundant. Can you delete them?